tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import csv
+import os
+from collections import defaultdict
+from itertools import islice
+from typing import Any, DefaultDict, Dict, Generator, List, Tuple, Union
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Catalog, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+_DATASET_NAME = "VGGFace2"
+_SEGMENTS = ("train", "test")
+
+
+_CLASS_ATTRS = ("Sample_num", "Flag", "Gender")
+
+_BOX2D_ATTRS = (
+    "01-Male",
+    "02-Black_Hair",
+    "03-Brown_Hair",
+    "04-Gray_Hair",
+    "05-Blond_Hair",
+    "06-Long_Hair",
+    "07-Mustache_or_Bread",
+    "08-Wearing_Hat",
+    "09-Eyeglasses",
+    "10-Sunglasses",
+    "11-Mouth_Open",
+)
+
+_KEYPOINTS = ("L_eye", "R_eye", "Nose", "L_mouth_corner", "R_month_corner")
+
+_BOX2D_FILES = (
+    os.path.join("label", "loose_bb_test.csv"),
+    os.path.join("label", "loose_bb_train.csv"),
+)
+_KEYPOINTS2D_FILES = (
+    os.path.join("label", "loose_landmark_test.csv"),
+    os.path.join("label", "loose_landmark_train.csv"),
+)
+
+_T = DefaultDict[str, Dict[str, Union[str, int, float, bool, List[Union[str, int, float, bool]]]]]
```

Comment:
we have a similar type `AttributeType` in `label/basic.py`, can we reuse it?

Reply:
I have read the file and it seems that AttributeType in it is not similar with the type I defined here.

Reply:
for `AttributeType`:
`AttributeType = Dict[str, Union[str, int, float, bool, List[Union[str, int, float, bool]]]]`

for your type:
`_T = DefaultDict[str, Dict[str, Union[str, int, float, bool, List[Union[str, int, float, bool]]]]]`

which means:
`_T = DefaultDict[str, AttributeType]`?